### PR TITLE
Improve scene

### PR DESCRIPTION
**Pull request by sir-marc**
This PR aims to make the difference between the optimized and non-optimized approaches meaningful on good machines as well.
image
image

### CODE REVIEW

File: benchmark/src/index.js [hunk 1]
```diff
@@ -9,13 +9,12 @@ import { generateHolisticReport } from "./reporter.js";
...
 const argv = yargs(hideBin(process.argv)).argv;
 
-const SAMPLE_TIMEOUT_MS = 20000;
-
 const LOG_DETAILS = argv.logDetails || false;
 setLogDetail(LOG_DETAILS);
 
 const main = async () => {
   const ITERATIONS = argv.iterations || 10;
+  const SAMPLE_TIMEOUT_MS = argv.timeout || 20000;
```
Comment (StuckiSimon) [inline benchmark/src/index.js:17]: new args shall be documented

File: demo/assets/airplane/lode-config.json [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
-    },
-    {
-      "threshold": 50,
-      "configuration": {
-        "targetScale": 0.125
-      }
+      "threshold": 300
     },
     {
       "threshold": -1,
```
Comment (StuckiSimon) [inline demo/assets/airplane/lode-config.json:7]: higher target scale does not work nicely?

File: demo/assets/head/license.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Model Information:
```
Comment (StuckiSimon) [inline demo/assets/head/license.txt:1]: have you considered the licenses for the other models as well?

File: demo/index.js [hunk 4]
```diff
@@ -21,86 +36,56 @@ const camera = new THREE.PerspectiveCamera(
...
-    name: "assets/apricot",
-    position: [-20, 0, -20],
-  },
-];
+const renderObjects = (scene, objs, selector = (obj) => obj) => {
+  objs.map(selector).forEach((obj, i) => {
+    for (let j = 0; j < config.objectCount; j++) {
+      const clone = obj.clone();
+      scene.add(clone);
+      clone.position.set(...generateRandomPosition(config));
```
Comment (StuckiSimon) [inline demo/index.js:51]: random positions have an effect on the used lods and therefore the performance changes "randomly"

**Review by StuckiSimon — COMMENTED** [review]
looks fine in general.
One important thing: the randomness makes the benchmark less reliable (e.g. frustum culling potentially has a big effect).
I strongly suggest to use a "seeded random approach".

File: .github/workflows/run-benchmark.yml [hunk 5]
```diff
@@ -43,4 +43,4 @@ jobs:
           cd benchmark
           yarn install --frozen-lockfile
           yarn run-demo
-          yarn start --iterations=5
+          yarn start --iterations=5 --logDetails --timeout 60000
```
Comment (StuckiSimon) [inline .github/workflows/run-benchmark.yml:46]: I suggest to disable logDetails again

File: demo/assets/airplane/lode-config.json (hunk 2 above)
Comment (sir-marc) [reply demo/assets/airplane/lode-config.json:7]: It's more about the change in color / lightning

Comment (StuckiSimon) [reply demo/assets/airplane/lode-config.json:7]: ⚡ 
you've added a bunch of new models and they have pretty high polycount - even when optimized. I'm just wondering whether you've tried it as we certainly have to tune it when running the benchmark.

File: demo/assets/head/license.txt (hunk 3 above)
Comment (StuckiSimon) [reply demo/assets/head/license.txt:1]: Do we have to write this in the report or somewhere related? (Also for the other models?) (As we'll include screenshots of the demo application)

**Review by StuckiSimon — APPROVED** [review]
Nice, that makes the benchmark more realistic 👍

**Pull request merged**